### PR TITLE
Fix the CKEditor scrolling issue 

### LINKED ISSUE #10737: Email editor scrolls/jumps down after every `enter`
### Mautic Version

4.0.0

### PHP version

7.4.26

### What browsers are you seeing the problem on?

_No response_

### What happened?

While editing long emails (MJML template), the editor jumps/scrolls down every time `enter` gets typed.

This isn't a problem if you don't have a vertical scroll bar or (strangely) if you are at the top of the page/email.
If you scroll a bit to edit text in the middle the editor starts to jump on every `enter`.

### How can we reproduce this issue?

1. Open the email editor of a long email
2. Scroll to the middle
3. Edit a text and hit `enter`

### Relevant log output

```shell
There is none
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by volha-pivavarchyk**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes [#10737](https://github.com/mautic/mautic/issues/10737)  

#### Description:
This PR fixes the bug a long email jumps/scrolls down when editing. 

This PR uses the way proposed [here](https://github.com/mautic/mautic/issues/10737#issuecomment-1157418764).

#### Steps to test this PR:
1. Open the email editor of a long email
2. Scroll to the middle
3. Edit a text and press enter

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the email editor of a long email
3. Scroll to the middle
4. Edit a text and press enter
5. The bug is gone

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11491"></a>

### CODE REVIEW

**Comment by RCheesley** [comment]
@volha-pivavarchyk I am not sure this has fixed the issue - I've spun up Gitpod with this PR applied, I've cleared the Mautic cache and my browser cache but I am still getting the editor jumping:

https://watch.screencastify.com/v/tOqhjLTjXPiTHJEYZTYu

**Comment by volha-pivavarchyk** [comment]
@RCheesley it looks like the style doesn't apply. 

That's a bit odd, because it worked fine for me locally and in Gitpod. I'll try testing it again to find the cause.

**Comment by T0mTom** [comment]
Hi,
for us this fix forked fine.
Thanks

**Comment by RCheesley** [comment]
Thanks for testing @T0mTom 🚀

**Comment by RCheesley** [comment]
@all-contributors please add @T0mTom for userTesting

**Review by RCheesley — DISMISSED** [review]
Re-tested this on Gitpod and it is now working - thanks @volha-pivavarchyk 🚀

File: plugins/GrapesJsBuilderBundle/Assets/library/js/grapesjs-editor.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+html, body, [data-gjs-type=wrapper] {
+    height: auto !important;
```
Comment (escopecz) [inline plugins/GrapesJsBuilderBundle/Assets/library/js/grapesjs-editor.css:2]: Is this necessary? It affect everything.

Comment (RCheesley) [reply plugins/GrapesJsBuilderBundle/Assets/library/js/grapesjs-editor.css:2]: @volha-pivavarchyk could you respond to the question here? Thanks!

Comment (volha-pivavarchyk) [reply plugins/GrapesJsBuilderBundle/Assets/library/js/grapesjs-editor.css:2]: @escopecz I added this style only into the builder iframe. It only affects the builder, and that's what the target is.

Comment (escopecz) [reply plugins/GrapesJsBuilderBundle/Assets/library/js/grapesjs-editor.css:2]: If it is enclosed in an iframe than it's ok. Thanks for explanation!

**Review by escopecz — APPROVED** [review]

**Pull request merged**